services/notifications/pkg/channels/channels.go

```diff
@@ -114,7 +114,12 @@ func (m Mail) SendMessage(ctx context.Context, message *Message) error {
 
 	email := mail.NewMSG()
 	if message.Sender != "" {
-		email.SetFrom(fmt.Sprintf("%s via %s", message.Sender, m.conf.Notifications.SMTP.Sender)).AddTo(message.Recipient...)
+		tmpSender := fmt.Sprintf("%s via %s", message.Sender, m.conf.Notifications.SMTP.Sender)
+		if strings.Contains(tmpSender, ",") {
+			tmpParts := strings.SplitN(tmpSender, "<", 2)
+			tmpSender = fmt.Sprintf("\"%s\" <%s", tmpParts[0], tmpParts[1])
+		}
```

Comment:
I think this is not yet robust enough. According to Wikipedia about Email format the space is also not allowed if the name is not inside "". 
Why not always try to split on "<" and if that works surround the first part by "" and take the second part in "<" as you do already?

That still does not support the case that the sender is a comma separated list of multiple addresses which seems to be valid as well, but not sure if that is applicable here.

Reply:
This function is setting the sender, the sender should never be more then one mail. I removed the if clause, now it will always add " around the sender name.